blackboxopt/optimizers/space_filling.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2020 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/boschresearch/blackboxopt
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import List
+
+try:
+    import numpy as np
+    from scipy.stats.qmc import Sobol
+except ImportError as e:
+    raise ImportError(
+        "Unable to import BOHB optimizer specific dependencies. "
+        + "Make sure to install blackboxopt[space-fill]"
+    ) from e
+from blackboxopt.base import MultiObjectiveOptimizer, Objective, SearchSpace
+from blackboxopt.evaluation import EvaluationSpecification
+
+
+class SpaceFilling(MultiObjectiveOptimizer):
+    """Sobol sequence based space filling optimizer.
+
+    Args:
+        search_space: The search space to optimize
+        objectives: The objectives of the optimization
+        seed: The sobol sequence is Owen scrambled and can be seeded for reproducibility
+    """
+
+    def __init__(
+        self, search_space: SearchSpace, objectives: List[Objective], seed: int = None
+    ) -> None:
+        super().__init__(search_space=search_space, objectives=objectives, seed=seed)
+        self._rng = np.random.default_rng(self.seed)
```

Comment:
Is there a particular reason to store the RNG here, too? We could also simply pass this instance to `Sobol`.

Reply:
No, actually went with just passing the seed along and not initializing an RNG at all. Thanks!